File: Curso Programação Orientada a Objetos/PHP com POO/Aula11/Bolsista.php
<?php

require_once 'Aluno.php';

class Bolsista extends Aluno {
    //Atributos
    private $bolsa;
    
    //Métodos
    public function renovarBolsa() {
        echo "<p>Renovando bolsa de " . $this->nome . "</p>";
    }
    
    //Método sobreposto
    public function pagarMensalidade() {
        echo $this->nome . " é bolsista! Pagamento facilitado";
    }
    
    //Getters
    public function getBolsa() {
        return $this->bolsa;
    }

    //Setters
    public function setBolsa($bolsa) {
        $this->bolsa = $bolsa;
    }


}

File: Curso Programação Orientada a Objetos/Java com POO/Aula11/src/aula11/Bolsista.java
package aula11;

public class Bolsista extends Aluno {
    //Atributos
    private float bolsa;
    
    //Métodos
    public void renovarBolsa() {
        System.out.println("Renovando bolsa de " + this.nome);
    }
    
    //Método sobreposto da classe-mãe Aluno
    @Override
    public void pagarMensalidade() {
        System.out.println(this.nome + " é bolsista! Pagamento facilitado");
    }
    
    //Getters e Setters
    //bolsa
    public float getBolsa() {
        return bolsa;
    }
    public void setBolsa(float bolsa) {
        this.bolsa = bolsa;
    }
    
}
